Add note about English-language only contributions (#8734)
braze-inc/braze-docs: _docs/_contributing/home.md

Some understanding of Git is required to contribute to Braze Docs. If you're new to Git and don't know where to start, see [Git Book: Getting Started](https://git-scm.com/book/en/v2/Getting-Started-About-Version-Control). If you just need a refresher, see [Git and GitHub]({{site.baseurl}}/contributing/git_and_github/).

{% alert note %}

> Community contributions are only accepted in English. English serves as the basis for translations into other supported languages.

Author: ```suggestion
We can only accept contributions in English. English serves as the basis for translations into other supported languages.
```
